src/crypto/crypto_ecdh.cc

```diff
@@ -443,7 +447,7 @@ Maybe<bool> ECDHBitsTraits::AdditionalConfig(
     return Nothing<bool>();
   }
 
-  params->id_ = GetCurveFromName(*name);
+  params->id_ = GetOKPCurveFromName(*name);
```

Comment:
AFAICT `id_` is only used to discern between X25519, X448 and then it throws everything else in the `default` bucket anyway, therefore returning the specific EC curve ids is not necessary.